src/LiquidationPairFactory.sol

```diff
@@ -2,49 +2,70 @@
 
 pragma solidity 0.8.17;
 
-import "openzeppelin/token/ERC20/IERC20.sol";
-
 import "./LiquidationPair.sol";
 
 contract LiquidationPairFactory {
+
+    /* ============ Events ============ */
     event PairCreated(
         LiquidationPair indexed liquidator,
         ILiquidationSource indexed source,
-        address target,
-        IERC20 indexed tokenIn,
-        IERC20 tokenOut,
+        address indexed tokenIn,
+        address tokenOut,
         UFixed32x9 swapMultiplier,
         UFixed32x9 liquidityFraction,
         uint128 virtualReserveIn,
         uint128 virtualReserveOut
     );
 
-    constructor() {}
+    /* ============ Variables ============ */
+    LiquidationPair[] public allPairs;
+
+    /* ============ Mappings ============ */
+
+    /// @notice tokenIn => tokenOut => pair
+    mapping(address => mapping(address => LiquidationPair)) public getPair;
```

Comment:
LiquidationPairs are, in a way, non-fungible.  You can create multiple pairs with the same in tokens and out tokens, but they may have different sources or multipliers or fractions.

Because of this, we shouldn't store the pairs in a mapping unique to (token in, token out)

Reply:
Removed: https://github.com/pooltogether/v5-liquidator/pull/2/commits/3c6df9ba1ef32975e87817268ffa0302215ec595